README.md

```diff
@@ -39,13 +39,20 @@ If you are using an old opam version, use the following commands to update it:
 	opam update
 	opam upgrade
 
-You can start the server for watching your file using this command and giving the path toyour MCore source file (this would be if your file is in the root directory of the project):
-	
-	cd src/ocaml-server/
-	dune exec ./main.exe -- path/to/source.mc [OPTIONAL -p PORT]
+You need to source the path to the IPM root folder as an environment variable:
+
+	export MI_IPM=/path/to/ipm
```

Comment:
Maybe add an example to make this more clear. Ex. "If you are in the root folder, the command will be export MI_IPM=./"

Reply:
Added this command as an example